src/amo/client/base.js

```diff
@@ -15,7 +14,6 @@ import tracking from 'amo/tracking';
 export default async function createClient(
   createStore,
   {
-    _FastClick = FastClick,
```

Comment:
This is referenced in the test setup for https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/client/test_base.js, so we should probably remove any code related to `FastClick` from that test file.